### PR TITLE
Add create subcommand

### PR DESCRIPTION
**Pull request by stefsmeets**
This PR adds a subcommand to create a file structure, copy and patch the template config file into it.

### CODE REVIEW

File: duqtools/create.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    template: DirectoryPath
+
+
+def create():
+    cfg = duqtools.config.Config()
+
+    options = cfg.create
+
+    template_drc = options.template
+    matrix = options.matrix
```
Comment (v1kko) [inline duqtools/create.py:44]: Nice!

File: duqtools/create.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+from pydantic import BaseModel, DirectoryPath
+
+import duqtools.config
+
+from .ids import write_ids
+from .jetto import JettoSettings
+
+
+class Sources(Enum):
```
Comment (v1kko) [inline duqtools/create.py:14]: Does it have to be an Enum? Good for now, but something to think about when/if we want to support other systems

File: duqtools/config.py [hunk 3]
```diff
@@ -16,6 +17,7 @@ class Config(BaseModel):
 
     # pydantic members
     submit: Optional[Submit_config] = Submit_config()
+    create: Optional[ConfigCreate]
```
Comment (v1kko) [inline duqtools/config.py:20]: No default values?

File: duqtools/__main__.py [hunk 4]
```diff
@@ -24,8 +20,9 @@ def cmdline():
                                           help='Create the UQ run files')
     parser_create.set_defaults(func=create)
 
-    parser_submit = subparsers.add_parser('submit', help='Submit the UQ runs')
-    parser_submit.set_defaults(func=submit)
+    parser_status = subparsers.add_parser(
```
Comment (v1kko) [inline duqtools/__main__.py:23]: You catched it as well :)

**Review by v1kko — COMMENTED** [review]
Merge and continue I would say! Let's get that prototype working

**Review by v1kko — APPROVED** [review]

File: duqtools/create.py (hunk 2 above)
Comment (stefsmeets) [reply duqtools/create.py:14]: Yeah, the other suggested alternative was `typing.Literal`, but that's not available in Python 3.7. I'd be happy if there was something else we could use.

File: duqtools/config.py (hunk 3 above)
Comment (stefsmeets) [reply duqtools/config.py:20]: Maybe later

**Pull request merged**